src/api/apiMethods.ts

```diff
@@ -0,0 +1,2 @@
+export const SEARCH_METHOD = 'flickr.photos.search';
```

Comment:
Create a single const object that is exported with search and info as its properties